ui/shared/ContactSelector.qml

```diff
@@ -0,0 +1,186 @@
+        }
+    }
+
+    Select {
+        id: select
+        label: ""
+        model: root.contacts
+        width: parent.width
+        menuAlignment: Select.MenuAlignment.Left
+        selectedItemView: Item {
+            anchors.fill: parent
+            Identicon {
+                id: iconImg
+                anchors.left: parent.left
+                anchors.leftMargin: 14
+                anchors.verticalCenter: parent.verticalCenter
+                height: 32
+                width: !!selectedContact.identicon ? 32 : 0
+                visible: !!selectedContact.identicon
+                source: selectedContact.identicon ? selectedContact.identicon : ""
+            }
+
+            StyledText {
+                id: selectedTextField
+                text: selectedContact.name
+                anchors.left: iconImg.right
+                anchors.leftMargin: 4
+                anchors.verticalCenter: parent.verticalCenter
+                font.pixelSize: 15
+                height: 22
+                verticalAlignment: Text.AlignVCenter
+            }
+        }
+        zeroItemsView: Item {
+            height: 186
+            StyledText {
+                anchors.fill: parent
+                text: qsTr("You don't have any contacts yet")
+                verticalAlignment: Text.AlignVCenter
+                horizontalAlignment: Text.AlignHCenter
+                font.pixelSize: 13
+                height: 18
+                color: Style.current.secondaryText
+            }
+        }
+
+        menu.delegate: menuItem
+        menu.width: dropdownWidth
+    }
+    TextEdit {
+        id: validationErrorText
+        visible: false
+        text: qsTr("Please select a contact")
+        anchors.top: select.bottom
+        anchors.topMargin: 8
+        selectByMouse: true
+        readOnly: true
+        font.pixelSize: 12
+        height: 16
+        color: Style.current.danger
+        width: parent.width
+        horizontalAlignment: TextEdit.AlignRight
+    }
+
+    Component {
+        id: menuItem
+        MenuItem {
+            id: itemContainer
+            property bool isFirstItem: index === 0
+            property bool isLastItem: index === contacts.rowCount() - 1
+
+            width: parent.width
+            height: visible ? 72 : 0
+            Identicon {
+                id: iconImg
+                anchors.left: parent.left
+                anchors.leftMargin: Style.current.padding
+                anchors.verticalCenter: parent.verticalCenter
+                width: 40
+                height: 40
+                source: identicon
+            }
+            Column {
+                anchors.left: iconImg.right
+                anchors.leftMargin: 12
+                anchors.verticalCenter: parent.verticalCenter
+
+                Text {
+                    text: name
+                    font.pixelSize: 15
+                    font.family: Style.current.fontBold.name
```

Comment:
I noticed this one is extremely "bold", but there also doesn't seem to be an actual design for that..

Reply:
I didn't look at the design nor what this actually looks like right now, but most of the time in the design, we just use `fonttWeight: medium`

Reply:
The design looks like this: 
![Imgur](https://imgur.com/ejIf2GR.png)
Have updated this to use `fontRegular.name` and `font.bold: true` instead.